src/appimagetool.c

```diff
@@ -903,6 +911,28 @@ main (int argc, char *argv[])
                         printf("Will not guess update information since zsyncmake is missing\n");
                     }
                 }
+	    } else if(GITHUB_REPOSITORY){
```

Comment:
Formatting is very broken in this entire block. Spaces set incorrectly around brackets, indentation is inconsistent. Needs to be fixed.

You want to compare against `NULL` here, so please be explicit and write it like that.

Reply:
I agree. I used `vim` to write the code, and I found out that my `vim` config is messed up. I realized it when I opened it in Vs Codium. Thanks!

Done on https://github.com/AppImage/AppImageKit/pull/1075/commits/bf7c4c5c9a8c2fbec53756475fd2c1a3055d64f2